### DISCUSSION TITLE
【建议】添加路径模拟功能

### DISCUSSION THREAD

**Discussion by ohmykreee** [ZCShou/GoGoGo#10] [Ideas]
~~软件写的非常好，孩子很喜欢，敏感肌也能用（误~~

非常喜欢这个软件，表示找了一圈就这个软件的功能最多，使用起更方便。

要是作者大大能增加一个路径模拟功能，能够预先设定模拟路径，然后直接开始模拟，这样一些常用的路径就不用每次都用摇杆手动去摇。（或者可以考虑录制模拟路径，然后在之后回放模拟路径，也是OK的）

**Comment by ZCShou** [comment]
这个可以有！

**Comment by haimichenha** [comment]
我也认为路径模拟可以有，双手双脚支持你
